### ISSUE TITLE
Security issue from `npm audit` with `lodash` < 4.17.5

### ISSUE THREAD

**Issue by svewag** [timkelty/shipit-utils#6]
Hi,

when I run `npm audit` in my project, I get a security issue warning for `lodash`.
At the moment `lodash ^3.5.0` is required. Is there a chance to update `lodash` to the something >=4.17.5?

It seems that you only use `lodash` for one situation.

```js
var _ = require('lodash');

/**
 * Test if all values are equal.
 *
 * @param {*[]} values
 * @returns {boolean}
 */

module.exports = function equalValues(values) {
  return values.every(function(value) {
    return _.isEqual(value, values[0]);
  });
};
```
https://github.com/shipitjs/shipit-utils/blob/9c074814b2d336df166423d14e6106c1373c64b8/lib/equal-values.js

Chances might be good that a higher version of `lodash` won't break this package.

Thank you.

**Comment by clement-escolano** [comment]
@neoziro What do you think about it ?

**Comment by svewag** [comment]
Any chance that the pull requests might be accepted?

**Comment by gregberge** [comment]
Solved in #7.

@svewag thanks for your work!

**Comment by gregberge** [comment]
Now published ;)
